dev/com.ibm.ws.kernel.service/src/com/ibm/ws/kernel/service/util/CpuInfo.java

```diff
@@ -445,21 +457,26 @@ public void run() {
             }
             // find available processors
             int runtimeAvailableProcessors = Runtime.getRuntime().availableProcessors();
-            int fileSystemAvailableProcessors = getAvailableProcessorsFromFilesystem();
+            float fileSystemAvailableProcessors = getAvailableProcessorsFromFilesystemFloat();
 
-            int newAvailableProcessors;
-            if (fileSystemAvailableProcessors <= 0 || fileSystemAvailableProcessors > runtimeAvailableProcessors) {
-                newAvailableProcessors = runtimeAvailableProcessors;
-            } else {
-                newAvailableProcessors = fileSystemAvailableProcessors;
-            }
+            int currentNumberOfProcessorsInt = AVAILABLE_PROCESSORS_INTEGER.get();
+            float currentNumberOfProcessorsFloat = AVAILABLE_PROCESSORS_FLOAT.floatValue();
```

Comment:
```suggestion
            float currentNumberOfProcessorsFloat = AVAILABLE_PROCESSORS_FLOAT.floatValue() / 100.0;
```

Reply:
I also think you should be consistent.  Either use get or floatValue.  Don't mix and match.